lib/forest_liana/bootstraper.rb

```diff
@@ -38,6 +38,10 @@ def generate_apimap
       create_apimap
       require_lib_forest_liana
       format_and_validate_smart_actions
+
+      unless Rails.env.production?
```

Comment:
`if Rails.env.development?` is a better condition.
What if the server is in `staging` environment or any other unknown type.